components/button/index.css

```diff
@@ -93,15 +131,17 @@ governing permissions and limitations under the License.
   color: inherit;
 
   .spectrum-Icon {
-    @inherit: %spectrum-ButtonIcon;
+    inset-inline-end: calc(var(--mod-spectrum-button-edge-to-visual, var(--spectrum-button-edge-to-visual)) - var(--mod-button-border-width, var(--spectrum-button-border-width)));
+    inset-block-end: calc(var(--mod-spectrum-button-edge-to-visual, var(--spectrum-button-edge-to-visual)) - var(--mod-button-border-width, var(--spectrum-button-border-width)));
```

Comment:
I think we should alias this calculation to its own custom property: it's the same for both properties (yes?) and it'll make things a bit cleaner.